tests/src/Nightwatch/Commands/performanceMarkEnd.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @file
+ * Ends last performance measurement mark.
+ *
+ * This provides a custom command, .performanceMarkEnd()
+ *
+ * @return {object}
+ *   The 'browser' object.
+ */
+
+exports.command = function performanceMarkEnd() {
+  const browser = this;
+
+  browser.performanceWaitBrowser().perform(() => {
+    let span = browser.apmSpans.pop();
+
+    if (!span) {
+      return;
+    }
+
+    span.end();
+
+    // Set spanId to current active span, if there is any.
+    span = browser.apmSpans.pop();
+
+    if (!span) {
+      return;
+    }
+
+    browser.setCookie({
+      domain: browser.apmDomain,
+      expiry: 3533274000,
```

Comment:
This is optional - do we really need it? Also why Thursday, 18 December 2081 09:00:00

Reply:
Yes, we do not need this option. I'll remove it.

And why `3533274000`, well that would be my 100th birthday. 😄